dev/dune-dbg.in

```diff
@@ -16,4 +16,11 @@ esac
 
 emacs="${INSIDE_EMACS:+-emacs}"
 
-ocamldebug $emacs $(ocamlfind query -recursive -i-format coq.top_printers) -I +threads -I dev $exe "$@"
+# Note: when doing a local dune build, ocamlfind won't work unless you do
+# export OCAMLPATH=<coq_root>/_build/install/default/lib
+# dune exports this automatically to child processes (e.g. this script).
+
+list_coq_packages="$(ocamlfind list | sed 's/ *(.*)//' | grep '^coq')"
+list_coq_include_args=$(ocamlfind query -recursive -i-format $list_coq_packages | sort | uniq)
+
```

Comment:
I need someone who knows more than me about `ocamldebug` to confirm this is OK; using `ocamlfind` here is a gross hack, and may fail if the build is not proper; I think we should instead add ocamldebug support to Dune.

Reply:
Well if there is a better way of doing it ... I checked that the include directories provided coincide with those directories where .cma files and sources reside. With the old solution a lot of paths were missing. So until there is a better way, I would use this as temporary solution.

Reply:
How about combining `ocamlfind -descendants coq.clib` with `ocamlfind -recursive coq.clib`? That should be fairly complete without sed manips.